tests/em/tdem/test_TDEM_crosscheck.py

```diff
@@ -161,17 +162,42 @@ def test_HJ_dhdtz_vtem(self):
         CrossCheck(prbtype1='MagneticField', prbtype2='CurrentDensity',
             rxcomp='MagneticFieldTimeDerivativex', waveform='vtem')
 
-
     def test_MagDipoleSimpleFail(self):
 
         print('\ntesting MagDipole error handling')
 
 
-        with warnings.catch_warnings(record=True):
+        with self.assertWarns(DeprecationWarning):
```

Comment:
I cannot figure out what ``error`` this was supposed to test, as it doesn't throw anything on `master`. It does throw some deprecation warnings though... @lheagy any thoughts? Did it not support an arbitrary dimension at some point?

Reply:
think I found it, had to go back a while. It would originally fail because the `receiver_list` was empty (or because it wasn't given a list of `receiver` object). Validation is handled by properties, and it being a 0 length list is actually okay (i.e. if we only want the fields).